std_msgs/Header header            # header info
string[] varNames
float64[] varValues
string[] varTypes

